packages/video_player/video_player/lib/video_player.dart

```diff
@@ -546,6 +555,22 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     await _applyPlaybackSpeed();
   }
 
+  /// Sets the caption offset of [this].
+  ///
+  /// [offset] indicates the delay in duration  The [offset] can be positive or negative.
+  /// the [offset] will be used when getting the correct caption for a specific position in [this]
+  ///
+  /// The values will be handled as follows:
+  /// *  0: this is the default behaviour. No offset will be applied.
+  /// * >0: The caption will have a positive offset. So you will get caption text from the future
```

Comment:
This seems backwards; I would expect a positive caption offset to move the captions later in the video. Is this based on a widespread standard?

Reply:
It makes more sense to change it. I have tested some other tools and indeed it is always done the way you stated.

Reply:
The implementation already did it like that. So the documentation was the only thing wrong here.